assets/js/util/index.js

```diff
@@ -53,6 +53,8 @@ export * from './standalone';
 export * from './storage';
 export * from './i18n';
 export * from './helpers';
+export * from './sort-object-map-by-key';
```

Comment:
Do we need to export these new functions? I thought the plan going forward was to keep util imports separated to make circular imports less likely and possibly do better chunk splitting. Since only new code references these files, we can scope the import to the specific files.